src/Chromely.CefSharp/Browser/Handlers/DefaultResourceSchemeHandler.cs

```diff
@@ -76,9 +76,14 @@ public override CefReturnValue ProcessRequestAsync(IRequest request, ICallback c
                         }
                     }
                 });
+                return CefReturnValue.ContinueAsync;
             }
-
-            return CefReturnValue.ContinueAsync;
+            else
+            {
+                StatusCode = (int)HttpStatusCode.NotFound;
```

Comment:
I'd suggest populating the `Stream` with an error message otherwise you'll end up with a blank page displayed to the user if they request a file that's not found.

You can use `ResourceHandler.GetMemoryStream` to turn a string into a stream.

https://github.com/cefsharp/CefSharp/blob/c1a4db20fde01a8464130384706993890838084a/CefSharp/ResourceHandler.cs#L333

Reply:
That is a good idea. 
Thanks.